bulbs/contributions/models.py

```diff
@@ -1,19 +1,127 @@
-from bulbs.content.models import Content
 from django.conf import settings
 from django.db import models
 
+from polymorphic import PolymorphicModel, PolymorphicManager
+
+from bulbs.content.models import Content, FeatureType
 
 # User = get_user_model()
 # User = get_model(*settings.AUTH_USER_MODEL.split("."))
 
+FLAT_RATE = 0
+FEATURETYPE = 1
+HOURLY = 2
+MANUAL = 3
+OVERRIDE = 4
+
+ROLE_PAYMENT_TYPES = (
+    (FLAT_RATE, 'Flat Rate'),
+    (FEATURETYPE, 'FeatureType'),
+    (HOURLY, 'Hourly'),
+    (MANUAL, 'Manual')
+)
+
+RATE_PAYMENT_TYPES = ROLE_PAYMENT_TYPES + ((OVERRIDE, 'Override'),)
+
+
+class FreelanceProfile(models.Model):
+    contributor = models.ForeignKey(settings.AUTH_USER_MODEL)
+    payment_date = models.DateTimeField(null=True, blank=True)
+
+    def get_pay(self, start=None, end=None):
+        qs = self.contributor.contributions.all()
+        if start:
+            qs = qs.filter(payment_date__gt=start)
```

Comment:
Especially since using only gt,lt would have miss some entries between to adjacent periods.
Ex: if two entries were for 12:00, and you did reports for (11:00-12:00) and (12:00-13:00), the 1200 entries would be filtered out of both.